app/src/main/res/layout/external_player_fragment.xml

```diff
@@ -58,15 +58,16 @@
 
 
         <ImageButton
-                android:id="@+id/butPlay"
-                android:layout_width="52dp"
-                android:layout_height="match_parent"
-                android:contentDescription="@string/pause_label"
-                android:background="?attr/selectableItemBackground"
-                app:srcCompat="?attr/av_play"
-                android:scaleType="fitCenter"
-                android:padding="8dp"
-                tools:src="@drawable/ic_av_play_white_48dp"/>
+            android:id="@+id/butPlay"
+            android:layout_width="52dp"
+            android:layout_height="match_parent"
+            android:contentDescription="@string/pause_label"
+            android:background="?attr/selectableItemBackground"
```

Comment:
Please do not change the indention of the file. Keep untouched lines like they were before.